### DISCUSSION TITLE
<details>/<summary> support

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3632] [Mobile] [answered]
i abuse these tags a lot to structurize issues and it looks especially horrible on mobile, when the screenshot/long code/log are fully displayed making you to scroll through them like crazy

**Comment by aconsuegra** [comment, marked as answer]
Hey @nt4f04uNd, thanks for your feedback!

Are you experiencing this issue on Android, iOS or both?

**Reply by nt4f04uNd** [reply]
only android. wasn't aware it is working on ios

**Reply by aconsuegra** [reply]
thanks for confirming! We are currently working on supporting this on Android too, so stay tuned for future updates!

**Reply by htcfreek** [reply]
Any updates when this will be supported?

**Reply by nt4f04uNd** [reply]
This was added in the recent update

**Reply by htcfreek** [reply]
> This was added in the recent update

Which version?

See this comment as an example where it didn't work: https://github.com/microsoft/PowerToys/pull/11303#issuecomment-841757900

**Reply by nt4f04uNd** [reply]
Dunno the exact version it was added, but I'm currently on v1.15.0-beta (10007)

Maybe it's only in beta

<img width="40%" src="https://user-images.githubusercontent.com/39104740/118531550-9fb98600-b74e-11eb-9c79-785b2fdb5669.jpg">

**Reply by aconsuegra** [reply]
That's correct. This is supported only in our Beta.

**Reply by htcfreek** [reply]
Update: It's now in stable for ANDROID released.
